AcademyApp/Views/Profile/LecturesListTableViewCell.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  LecturesListTableViewCell.swift
+//  AcademyApp
+//
+//  Created by Peter Sevcik on 09/07/2019.
+//  Copyright © 2019 Peter Sevcik. All rights reserved.
+//
+
+import UIKit
+
+class LecturesListTableViewCell: UITableViewCell, NibLoadable {
```

Comment:
Name LecturesListTableViewCell suggests that there are multiple lectures in the cell, I'd name it LectureTableViewCell or similar